code/modules/mob/mob.dm

```diff
@@ -446,9 +446,31 @@
 		// shift-click catcher may issue examinate() calls for out-of-sight turfs
 		return
 
-	if(is_blind())
-		to_chat(src, "<span class='warning'>Something is there but you can't see it!</span>")
-		return
+	if(is_blind()) //blind people see things differently (through touch)
+		//need to be next to something, awake, on help intent, and have an empty hand
+		if(!in_range(A, src) || incapacitated() || a_intent != INTENT_HELP || LAZYLEN(do_afters) >= get_num_arms() || get_active_held_item())
```

Comment:
Help intent seems like an unintuitive requirement for this.

Reply:
I used help intent only so you wouldn't punch or shove mobs you were examining. Could be removed, I suppose

Reply:
I think you should be able to set the intent before attacking, then restore it after.